### PR TITLE
Fixes #9135: Improve technical and event log datepicker UX

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/9135

### CODE REVIEW

File: rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js [hunk 1]
```diff
@@ -1600,20 +1600,20 @@ function createTable(gridId,data,columns, customParams, contextPath, refresh, st
...
 
     var pickEndInput = $('<input type="text">');
     pickEndInput.datetimepicker({dateFormat:'yy-mm-dd', timeFormat: 'HH:mm:ss', timeInput: true});
-    pickEndInput.addClass("pickEndInput");
-    $("#"+gridId+"_wrapper .dataTables_pickend").append(pickEndInput);
+    pickEndInput.addClass("pickEndInput form-control input-xs");
+    $("#filterLogsEnd").append(pickEndInput);
 
-    var pickDatesButton = $("<button>Get Events Between Dates</button>");
+    var pickDatesButton = $("<button>Get Events</button>");
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js]: Maybe we could have the button directly in html and only add the click function to "#filterLogsButton"

What do you think @RaphaelGauthier ?

Comment (RaphaelGauthier) [inline rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js]: Good idea !

**Comment by RaphaelGauthier** [comment]
PR updated

File: rudder-web/src/main/webapp/templates-hidden/node_logs_tabs.html [hunk 2]
```diff
@@ -2,7 +2,13 @@
   <h3>Technical logs from Directives applied to this node</h3>
   <hr class="spacer"/>
   <p class="reportsHeader">The table below contains all the technical logs received from this node. These are used to calculate Compliance (see previous tab), and may be useful in analyzing problems, or obtaining details in the case of automatic repairs.</p>
-
+  <div id="filterLogs" class="tw-bs reportsHeader">
+    Get technical logs between
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/templates-hidden/node_logs_tabs.html]: Can you replace 'technical logs' by 'reports' (in button too)

File: rudder-web/src/main/webapp/secure/utilities/eventLogs.html [hunk 3]
```diff
@@ -14,6 +14,13 @@
     <div class="lift:administration.EventLogsViewer.display">
     </div>
     <span id="eventLogsError"></span>
+    <div id="filterLogs" class="tw-bs space-top hide">
+      Get event logs between
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/secure/utilities/eventLogs.html]: Replace 'event logs' by 'events'

**Comment by RaphaelGauthier** [comment]
PR updated

**Comment by RaphaelGauthier** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**